### ISSUE TITLE
The hotkey for T+>> should always be available

### ISSUE THREAD

**Issue by felixvd** [matecat/MateCat#39]
The "T+>>" button is only available when the next segment is translated. When it is not, the hotkey does not work, although it should, since the user input "This is translated, take me to whatever the next segment is" still makes sense.

Currently, the user has to manually parse if the next segment is translated and select the right button/press the right hotkey to proceed to the next untranslated segment. If you mess up, you either pressed a hotkey for nothing (bad), or you are taken to a segment that has already been translated (bad). There is no reason to make the user do this.

The best solution in my opinion is to keep the T+>> button visible at all times. Sometimes its behavior will be identical to the last one, but since it will not confuse or overload the interface, that should be no problem.

**Comment by marcotrombetti** [comment]
Thanks Felix,

T+>> is visible when you can actually jump to a untranslated segment that
is not the next one. This keep the UI clean with only one button for most
of the time. The button appears when you need it: when yuo start doing a a
review or when you go back to fix random segments.

From your email I did not get if
1) we don't do what I described (bug)
2) you want a different behaviour. In case can you explain why?

Marco

**Comment by felixvd** [comment]
Hi Marco,

The interface works as you described, but I believe that it is suboptimally designed. I edited the original post a little for clarity, and I will explain some more below.

As long as there are untranslated segments, a translator will almost always want to pass to the next untranslated segment. However, as translated segments auto-propagate through the document, there is no single hotkey to do this. Instead, the user has to check manually after every segment which hotkey to press, or they get an undesirable outcome: either the segment selection does not advance (Cmd+Shift+Enter when T+>> was disabled) or the selected segment is now already translated (Cmd+Enter when T+>> was available). 

Consider [this screenshot of a section][1] of my current job. To progress through it with minimal friction, a different hotkey has to be pressed after every segment. This kind of manual checking is unnecessary and should be automatic.

In summary: Ctrl/Cmd+Shift+Enter should *always* take you to the next untranslated segment, even if that segment is the next one. You may hide the button for design reasons, but you should allow its hotkey to be used always.

[1]: http://imgur.com/DNV1Hzv

**Comment by Ostico** [comment]
Old question.

Close.
